soc/arm/nxp_imx/rt/Kconfig.soc

```diff
@@ -99,14 +99,15 @@ config SOC_MIMXRT1062
 config SOC_MIMXRT1064
 	bool "SOC_MIMXRT1064"
 	select HAS_MCUX
+	select HAS_MCUX_CACHE
 	select HAS_MCUX_CCM
 	select HAS_MCUX_ELCDIF
 	select HAS_MCUX_ENET
 	select HAS_MCUX_IGPIO
 	select HAS_MCUX_LPUART
 	select HAS_MCUX_TRNG
 	select CPU_HAS_FPU_DOUBLE_PRECISION
-	select CPU_HAS_MPU
+	select CPU_HAS_ARM_MPU
```

Comment:
This change isn't needed after commit ee3f7d629b9d3f2fae06351bfa0e2dfece1cf935

Reply:
ok thanks